WireAnalytics/Package.swift

```diff
@@ -0,0 +1,58 @@
+// swift-tools-version: 5.10
+
+import Foundation
+import PackageDescription
+
+let package = Package(
+    name: "WireAnalytics",
+    platforms: [
+        .iOS(.v15),
+        .macOS(.v12)
+    ],
+    products: [
+        .library(
+            name: "WireAnalytics",
```

Comment:
Don't we have to embed/link these libraries into Wire-iOS?

Reply:
No, because it's linked to `WireCommonComponents`, that are linked to Wire-iOS.